Robot2018/src/org/usfirst/frc/team199/Robot2018/commands/RunScript.java

```diff
@@ -55,7 +55,7 @@ public RunScript(String scriptName) {
 	    			case "end":
```

Comment:
I'm a little confused as to how this "jump" option works, esp w/ calling the RunScript command recursively. From what I understand, the parameter you're passing into RunScript again - should this case be reached - is the second "word" (i.e. second unbroken String) in the first command/line of the chosen script. Normally, for example in the case of a "moveto" being the first word, the second word would be an extra parameter. In the "jump" case, however, would this second word be the name of another script (i.e. another 4 character script name like CLRx)? If my assumptions are correct, this "jump" case would be used in the event that the script name originally entered was impossible, henceforth calling this command again, making the recursive script name parameter a script name that does nothing or just crosses the baseline? Is this where you going with the "jump" option? But then again wouldn't the "NOTHING" strategy enum option in Autonomous (before RunScript is run) take care of impossible options? I'm just a little confused. :)

Reply:
No, the jump command is there to avoid repetition in the scripts. For example, if there are two scripts that both need to grab a cube from the left side of the switch and eject it to the right scale at one point, you would have a script that does that, and each of the two original scripts would jump to that script when they want to do that